packages/cmake/project.bri

```diff
@@ -184,7 +185,9 @@ export function cmakeBuild(
   }
 
   return std.runBash`
-    export LIB="$LIBRARY_PATH"
+    # Set CMake search hints to find dependencies
+    export CMAKE_INCLUDE_PATH="$CPATH"
```

Comment:
Those two environments variables are used by the CMake macros to tweak the search hint paths.